themes/codefor-theme/layouts/shortcodes/button-center.html

```diff
@@ -1,5 +1,7 @@
+
+{{ $target := .Get "target" }}
 <div class="d-flex justify-content-center">
-    <a class="btn btn-outline-dark {{ .Get "class" }} " href="{{ .Get "href" }} ">
+    <a class="btn btn-outline-dark {{ .Get "class" }} " href="{{ .Get "href" }} " {{ if $target }}target="_blank"{{ end }}>
```

Comment:
links mit `target="_blank"` sollten auch `rel="noopener noreferrer"` haben. Dies hier ist nicht die einzige Stelle.
```suggestion
    <a class="btn btn-outline-dark {{ .Get "class" }} " href="{{ .Get "href" }} " {{ if $target }}target="_blank" rel="noopener noreferrer"{{ end }}>
```